workshops/react_calendar/README.md

```diff
@@ -0,0 +1,670 @@
+eg: If the filename is `Abc.js`, the component's name will be `Abc`.
+
+4. Return jsx.
+
+5. At the very end or before the function's name, export the function.
+
+6. Import the component in the `App.js` / `index.js` file.
+
+Next, open the file which you just created and let's write our first React code.
+
+We'll first import `React` and `components.css` in this manner:
+```jsx
+import React from 'react';
+import './components.css';
+```
+
+**Note:** The `./` on line 2 means that the file which we are importing is in the same directory.
+
+Next, we'll create a functional component. You can create it the normal way or by following the ES6 syntax. Here's how you'll do it either way.
+
+```jsx
+function CalendarUI() {
+  return()
+}
+```
+
+OR
+
+```jsx
+const CalendarUI = () => {
+  return()
+}
+```
+
+Use any syntax you like or feel that it is suitable/easier for you to understand.
+
+Now we will return jsx code in our functional component. One remarkable thing to keep in mind is that we can't return more than 1 parent elements in our code. So, we'll always need to create a main parent `<div>` tag and then write all the code inside of that.
+
+**This is wrong:**
+```jsx
+return(
+  <h1>Hello world</h1>
+  <h1>Lol</h1>
+  <h1>Hi!!</h1>
+)
+```
+
+**This is the correct way:**
+```jsx
+return(
+  <div>
+    <h1>Hello world</h1>
+    <h1>Lol</h1>
+    <h1>Hi!!</h1>
+  </div>
+  // Don't write anything outside the parent element.
+)
+```
+
+Now let's write the following jsx code (inside the `return` parenthesis).
+
+```jsx
+<div className="container">
+  <div className="main">
+    <p className="month"></p>
+    <p className="day"></p>
+    <p className="date"></p>
+    <p className="year"></p>
+  </div>
+</div>
+```
+
+We won't see anything in our preview now, because our `<p>` tags are empty. By looking at the `className` you might have understood what will come there. Let's hardcode today's date in it and let's check our preview window!
+
+Wait what? Nothing on the preview window? It's because we haven't followed our 5th and the 6th step yet! 
+
+It is a common mistake to forget this step while writing React, so if you ever feel your code isn't working fine, check your imports and exports first!
+
+At the very end of our file, we'll export the component.
+```jsx
+export default CalendarUI;
+```
+
+Your code so far:
+```jsx
+import React from 'react';
+import './components.css';
+
+const CalendarUI = () => {
+  return(
+    <div className="container">
+      <div className="main">
+        <p className="month">October</p>
+        <p className="day">Sunday</p>
+        <p className="date">18</p>
+        <p className="year">2020</p>
+      </div>
+    </div>
+  )
+}
+
+export default CalendarUI;
+```
+
+We can also export it without the `default` keyword but for today's sake, we'll use the `default` keyword. Now we also need to import it somewhere so that it is rendered. Can you guess where will we import it? Yes! It's the `App.js` file.
```

Comment:
"today's sake" is awkward phrasing, maybe something like "for the scope of this workshop" or something like that